src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -313,15 +310,6 @@ class ClusterMonitorActor(val cluster: Cluster,
     } yield result
   }
 
-  private def isProxyAvailable(clusterStatus: ClusterStatus, ip: IP): Future[Boolean] = {
-    for {
-      _ <- ensureClusterReadyForProxying(ip, clusterStatus)
-      proxyAvailable <- jupyterProxyDAO.getStatus(cluster.googleProject, cluster.clusterName)
-    } yield {
-      proxyAvailable
-    }
-  }
```

Comment:
Removed the method above and renamed `JupyterProxyDAO.getStatus` as `isProxyAvailable` and started calling that here directly because

1. with the call `ensureClusterReadyForProxying(ip, clusterStatus)` no longer necessary, this method would've been down to the `jupyterProxyDAO.getStatus` call
2. `JupyterProxyDAO.getStatus` was not returning us a `status` per se but whether the proxy is available or not